test/steps/kogitoapp.go

```diff
@@ -179,7 +179,18 @@ func (data *Data) scaleKogitoApplicationToPodsWithinMinutes(name string, nbPods,
 	if err != nil {
 		return err
 	}
-	return framework.WaitForDeploymentConfigRunning(data.Namespace, name, nbPods, timeoutInMin)
+
```

Comment:
I realized that this change in `scaleKogitoApplicationToPodsWithinMinutes` can be removed as KogitoRuntime scaling is done in its own function, not here.

Reply:
Reverted